hack/testing/logging.sh

```diff
@@ -123,97 +131,79 @@ os::log::system::start
 
 export KUBELET_HOST=$(hostname)
 
-os::start::configure_server
-if [ -n "${KIBANA_HOST:-}" ] ; then
-    # add loggingPublicURL so the OpenShift UI Console will include a link for Kibana
-    # this part stolen from util.sh configure_os_server()
-    cp ${SERVER_CONFIG_DIR}/master/master-config.yaml ${SERVER_CONFIG_DIR}/master/master-config.orig.yaml
-    openshift ex config patch ${SERVER_CONFIG_DIR}/master/master-config.orig.yaml \
-              --patch="{\"assetConfig\": {\"loggingPublicURL\": \"https://${KIBANA_HOST}\"}}" > \
-              ${SERVER_CONFIG_DIR}/master/master-config.yaml
+if [ $NOSETUP = 1 ] ; then
+    echo skipping openshift setup and start
+else
+    os::start::configure_server
```

Comment:
@jcantrill You are correct - this does _does not make CI work_.  It only allows the logging CI tests to be run manually against a 1.4/3.4 install.